src/js/stores/MesosStateStore.js

```diff
@@ -304,4 +306,27 @@ class MesosStateStore extends GetSetBaseStore {
   }
 }
 
+if (Config.useFixtures) {
+  const getMasterFixture = require("../../../tests/_fixtures/v1/get_master.json");
+  const subscribeFixture = require("../../../tests/_fixtures/v1/subscribe.js");
+
+  const { MesosStreamType } = require("../core/MesosStream");
+
+  const oldRequest = request;
+
+  request = function(options, url) {
+    if (url === "/mesos/api/v1?get_master") {
+      console.log("Stub /mesos/api/v1?get_master");
```

Comment:
Is this a wanted console.log?

Reply:
it's a fixture mode only, we can totally remove the log